actors/runtime/src/main/java/cloud/orbit/actors/runtime/ActorProfiler.java

```diff
@@ -44,7 +43,7 @@
 {
     private final HashMap<Object, ProfilerData> collectedData = new HashMap<>();
     private final ReadWriteLock rwlock = new ReentrantReadWriteLock();
-    private final KryoCloner kryoCloner = new KryoCloner();
+    private final KryoSerializer kryoCloner = new KryoSerializer();
```

Comment:
Looks like the ActorProfiler is pretty old from 2015 and and isn't actually used anywhere. It can probably be removed, we have better ways to track execution time of tasks now with invocation handler extensions etc